lib/internal/websockets/BufferUtil.js

```diff
@@ -0,0 +1,50 @@
+/*!
+ * ws: a node.js websocket client
+ * Copyright(c) 2011 Einar Otto Stangvik <einaros@gmail.com>
+ * MIT Licensed
+ */
+function BufferUtil() {}
```

Comment:
This can be a `class`. Although since it carries no state I'm not sure why it's not just a sealed object. What's the benefit of this?